app/src/main/java/com/github/se/wanderpals/model/data/TripNotification.kt

```diff
@@ -7,12 +7,13 @@ import java.time.LocalDateTime
  * be triggered by various events related to the user's travel itinerary.
  *
  * @param title A brief title of the TripNotification.
- * @param path A string path that could represent the itinerary path or related resource.
+ * @param route The route of the navigation (can be empty).
  * @param timestamp The exact date and time when the TripNotification was created.
+ * @param navActionVariables The navigation action variables stored as a string.
  */
 data class TripNotification(
     val title: String,
-    val path: String,
+    val route: String,
```

Comment:
I don't think this change was needed, it makes some files are changing for no reason which makes this PR bigger. Maybe in the future, see if it's really relevant to do it. Just an opinion

Reply:
Yes you are right, is it just that path makes less sens than route so I wanted to refactor it since notification will probably be used later for other purpose (e.g. finance screen) so I decided to already make the refactoring